app/controllers/users_controller.rb

```diff
@@ -27,17 +26,33 @@ def update
   def create
     @user = User.new(user_attributes)
 
-    if current_user && current_user.admin?
-      @user.role_id = params[:user][:role_id]
-    end
+    @user.role_id = params[:user][:role_id] if current_user && current_user.admin?
 
     if @user.save
-      redirect_to users_path, :notice => 'User added'
+      redirect_to users_path, notice: 'User added'
     else
       render :new
     end
   end
 
+  def resend_confirmation_email
+    user = User.find_by(id: params[:id])
+
+    if user.nil?
+      redirect_to users_path
+      flash[:notice] = 'User does not exist.'
+      return
```

Comment:
Usually controllers do not contain a `return`, but rather a `render` or a `redirect`